gut_to_soil_manuscript_figures/scripts/plot_pcoa_2d.py

```diff
@@ -73,17 +73,16 @@ def _bucket_util(highlighted_buckets, md, ord_2d):
         # bulking
         bucket_ids_bulk_week0 = \
             md[(md['Bucket'] == bucket) &
-               (md['SampleType2'] == 'Bulking Material') &
-               (md['Week'] == 0.0)].index.values
+               (md['SampleType'] == 'Bulking Material')].index.values
```

Comment:
Same comment as above, it looks like `(md['Week'] == 0.0)].index.values` was removed here as well.